### PR TITLE
add pentration testing

### PR DESCRIPTION
**Pull request by mheriyanto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ Collection of awesome Python resources for testing and generating test data.
 ## Mutation Testing
 
 - [Cosmic Ray](https://github.com/sixty-north/cosmic-ray) - makes small changes to your source code, running your test suite for each one.
+- [Mutatest](https://mutatest.readthedocs.io/en/latest) - Python mutation testing.
```
Comment (cleder) [inline README.md:92]:
```suggestion
- [Mutatest](https://github.com/EvanKepner/mutatest) - Python mutation testing.
```

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Collection of awesome Python resources for testing and generating test data.
 
 ## Code Coverage
 
-- [coverage](https://pypi.org/project/coverage) - Code coverage measurement.
+- [Coverage.py](https://coverage.readthedocs.io) - is a tool for measuring code coverage of Python programs.
```
Comment (cleder) [inline README.md:45]:
```suggestion
- [Coverage.py](https://github.com/nedbat/coveragepy) - is a tool for measuring code coverage of Python programs.
```

**Review by cleder — CHANGES_REQUESTED** [review]
I think we should have the (GitHub) repository link as standard

File: README.md (hunk 1 above)
Comment (mheriyanto) [reply README.md:92]: Okay, I agree to be the (GitHub) repository link as standard

**Pull request merged**